scripts/mk_util.py

```diff
@@ -1694,10 +1716,11 @@ def mk_makefile(self, out):
             key = "<AssemblyOriginatorKeyFile>%s</AssemblyOriginatorKeyFile>" % self.key_file
             key += "\n<SignAssembly>true</SignAssembly>"
 
-        version = get_version_string(3)
+        version = get_version_string(4)
 
-        core_csproj_str = """<Project Sdk="Microsoft.NET.Sdk">
+        print(f"Version output to csproj: {version}")
```

Comment:
sorry, but we can't use formatting with 'f' in our build scripts due to down-level python under some platforms.